UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/RootBodyPartContainer.cs

```diff
@@ -381,6 +381,28 @@ public virtual void SubBodyPartAdded(BodyPart implant)
 			}
 		}
 
+		/// <summary>
+		/// Applies slash damage to limbs inside of this container. Armor chance is handled inside of ApplyCutSizeLogic();
+		/// </summary>
+		public void TakeSlashDamage(float damage)
+		{
+			foreach (var ContainsLimb in ContainsLimbs)
```

Comment:
camelCase. Also if you can find another name that better represents the element, I was confused by how similar it sounds to the list.